### PR TITLE
Fix link to the shop on backoffice

### PR DESCRIPTION
**Pull request by PatrykMakowski**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Invalid link to store if the store is on a different domain than currently used in backoffice.
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | 
| How to test?  | Two shops in multistore with a different domains. Link in the navigation should be with different domain for each shop. 

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/12410)

### CODE REVIEW

File: classes/shop/Shop.php [hunk 1]
```diff
@@ -703,6 +703,8 @@ public static function cacheShops($refresh = false)
                 }
 
                 $row = $row + array('theme_name' => '');
+				$domain_with_protocol = (Configuration::get('PS_SSL_ENABLED', null, $row['id_shop_group'], $row['id_shop'])
```
Comment (PierreRambaud) [inline classes/shop/Shop.php:706]: `Tools::getShopProtocol` exists to get you the correct protocol

Comment (PatrykMakowski) [reply classes/shop/Shop.php]: Yes, but only for current shop.

Comment (PatrykMakowski) [reply classes/shop/Shop.php:706]: Yes, but this get value always for current shop in context.

**Comment by PierreRambaud** [comment]
Really sorry for the dealy, are you able to rebase your branch?

**Comment by matks** [comment]
Hi, [following our review guidelines](https://devdocs.prestashop.com/1.7/project/maintainers-guide/reviewing-pull-requests/) "Pull Requests may be closed after 30 days of inactivity following a request for modifications".

Since we did not get answer from author I close this PR.

Please note this does not mean it's a final "won't merge" 😉 if you come back we can reopen it and update it

Thank you for the contribution

**Pull request closed without merging**